Tests/mock_server.py

```diff
@@ -364,6 +364,9 @@ def start(self, playbook_id, path=None, record=False, thread_index=0, prints_man
             self.repo_folder, get_folder_path(playbook_id), 'problematic_keys.json'
         )
         current_problem_keys_filepath = os.path.join(path, get_folder_path(playbook_id), 'problematic_keys.json')
+        silence_output(
```

Comment:
let's add the explanation in a comment, as it is not easy to interpret from code flow.

Reply:
This way the `problematic_keys.json` will not be empty in the pwned v2 case that we saw, so we will not reach the error. right?

Reply:
We wouldn't reach it for two reasons.
A. It wouldn't be empty as you stated.
B. Now we only call `proxy.clean_mock_file` if `run_test_logic` succeeds - so in this particular case, with my update, `proxy.clean_mock_file` wouldn't be called at all.